packages/components/src/atoms/Price/Price.tsx

```diff
@@ -53,8 +53,8 @@ const Price = forwardRef<Omit<HTMLSpanElement, 'children'>, PriceProps>(
     },
     ref
   ) {
-    const formattedPrice = formatter(value, variant)
-
+    const formatedValue = Number(String(value * 100).split('.')[0])/100
```

Comment:
why do you multiply by 100 and then divide by 100, is it unambiguous?

Reply:
@emersonlaurentino no because I´m formatting the number I want to truncat and after that I´m formating to the price format expected. This is necessary because otherwise the price won´t follow the checkout logic showing 3,97 in the example I sent for instance.

Example 21 items: 
value = 3.4676
value * 100 = 346.76
String(value * 100).split('.')[0] = 346
Number(String(value * 100).split('.')[0])/100 = 3.46


value = 3.4676
String(value).split('.')[0] = 3
Number(String(value).split('.')[0]) = 0.03

Reply:
That's a formatting issue. You should use the formatter function to correct that.

Reply:
Exactly @beatrizmaselli you should use the formatter function for that.

Reply:
Nice! This is actually not needed so I removed :)